lib/src/ui/movie_detail.dart

```diff
@@ -7,9 +7,9 @@ import '../models/result_movie.dart';
 class MovieDetail extends StatelessWidget {
   final ResultMovie result;
```

Comment:
In this case instead of using all the result object you can use it only the parameters that you need. You only need: posterPath, title, releaseDate, voteAverage, overview, originalLanguage, voteCount

Reply:
Done, movie_detail file now receive that parameters from movie_grid_builder instead of passing all the ResultMovie object. Also the movie_detail_test file needed to be modified.